tests/run.sh

```diff
@@ -3,36 +3,14 @@ set -eo pipefail
 
 : "${SERVER:=http://localhost:8888/v1}"
 
-usage() {
-    echo "usage: ./run.sh integration-test|browser-test"
-    echo ""
-    echo "    integration-test        Start integration tests"
-    echo "    browser-test            Start browser tests"
-    echo ""
-    exit 1
-}
-
-[ $# -lt 1 ] && usage
-
 wait_for_server () {
   echo "Waiting for $SERVER to become available"
   wget -q --retry-connrefused --waitretry=1 -O /dev/null $SERVER || (echo "Can't reach $SERVER" && exit 1)
   echo "verifying $SERVER heartbeat"
   http --check-status --body --json --pretty format GET $SERVER/__heartbeat__ ; echo
 }
 
-case $1 in
-integration-test)
-    shift
-    wait_for_server
-    pytest integration $@
-    ;;
-browser-test)
-    shift
-    wait_for_server
-    pytest browser_test.py --log-level=DEBUG --browser firefox
-    ;;
-*)
-    exec "$@"
```

Comment:
Are we talking about run.sh in general? Or something else?

Reply:
You removed the abitilty to use `run.sh` with abritary params/commands, that's why I was wondering if we, as developers debugging local containers or whatevers, were using this capability

Reply:
Oh, the exec line specifically. I didn't fund any uses for it but I'm happy to put that back if we want to keep it just in case. If I wanted to run customized tests locally I'd just modify run.sh or just execute pytest myself, but maybe not everybody works that way.

Reply:
Let's ask @grahamalama ! I'm fine with anything